src/core_ocean/analysis_members/mpas_ocn_surface_area_weighted_averages.F

```diff
@@ -350,9 +350,10 @@ subroutine ocn_compute_surface_area_weighted_averages(domain, timeLevel, err)!{{
          ! copy data into work array
          ! Note: Order of indices must match the ordering of vars in var_arrays in Registry_surface_area_weighted_averages.xml
          !       (and all var_arrays in Registry_surface_area_weighted_averages.xml must have same ordering!)
+
+         !initialize workArray to zeros
+         workArray(:,:) = 0.0_RKIND
          n = 1
-         workArray(n,:) = 0.0_RKIND
```

Comment:
good catch @vanroekel. But didn't the `n=1` on the previous line help initialize `workArray(n,:)` correctly?

Reply:
it only initialized the 1st index, previously all indices were initialized to zero.  I've reverted now to using
```
workArray(:,:) = 0.0_RKIND
```
which should initialize the whole array to zero and not just workArray(1,:)

Reply:
I see. I don't even remember when this analysis member was changed. Anyways, thanks for finding this.